_rules/element-lang-valid-de46e4.md

```diff
@@ -184,7 +184,7 @@ This `article` element has a `lang` [attribute value][] which consists of only [
 
 #### Failed Example 4
 
-The `lang` [attribute value][] does not have a valid language tag. The `lang` attribute must be valid because the content is [visible][].
+The `lang` [attribute value][] does not have a valid language tag. Even though its descendant `p` element is not [included in the accessibility tree][] due to the presence of the `aria-hidden="true"` attribute, the content of the `p` element is still [text inheriting its programmatic language][] from the `article` element. Hence, the lang attribute must be valid.
```

Comment:
```suggestion
The `lang` [attribute value][] does not have a valid language tag. Even though the `p` element is not [included in the accessibility tree][] due to the presence of the `aria-hidden="true"` attribute, it is [visible][]; therefore its content is [text inheriting its programmatic language][] from the `article` element. Hence, the lang attribute must be valid.
```
(there is only one `p` element, so no need to disambiguate, plus "its (descendant)" refers here to the `lang` attribute (value) (subject of the previous sentence) making "its descendant" technically incorrect)